components/HabitList.js

```diff
@@ -0,0 +1,22 @@
+import styled from "styled-components";
+import { HABITS } from "../lib/db";
+
+function Card({ habit }) {
+  return (<ul>{habit.id}</ul>), (<p>{habit.name}</p>);
```

Comment:
this doesn't seem correct, a ul with only text in it, and you try to return 2 elements inside the card?

Reply:
is it fine if I export the function as Habit to index.js with single element?

Reply:
> this doesn't seem correct, a ul with only text in it, and you try to return 2 elements inside the card?

is it fine if I export the function as Habit to index.js with single element?